@utahdts/utah-design-system/react/components/forms/FileInput/FileTemplate.jsx

```diff
@@ -0,0 +1,33 @@
+import { useContext } from 'react';
+import { isFunction } from 'lodash';
+import { FileContext } from './context/FileContext';
+import { joinClassNames } from '../../../util/joinClassNames';
+
+/**
+ * @param {object} props
+ * @param {import('react').ReactNode} props.children
+ * @param {string} [props.className]
+ * @returns {import('react').JSX.Element}
+ */
+export function FileTemplate({
+  children,
+  className,
+}) {
+  const fileContext = useContext(FileContext);
+
+  let content;
+  if (isFunction(children)) {
+    // @ts-ignore
+    content = children(fileContext);
+  } else if (children) {
```

Comment:
children is required in the JSDoc so it can't be "falsey". Even if children wasn't required, `content` could just be set to equal `children` since rendering something is the same as rendering null/undefined:
```
} else {
  content = children;
}
```
And this may be a good use of a ternary in the return?
```
<li ...>
  {isFunction(children) ? children(fileContext) : children
</li>
```

Reply:
Yes, remove it.